packages/react-native-reanimated/src/layoutReanimation/web/componentUtils.ts

```diff
@@ -21,14 +21,26 @@ import type { ReanimatedSnapshot, ScrollOffsets } from './componentStyle';
 import { setElementPosition, snapshots } from './componentStyle';
 import { Keyframe } from '../animationBuilder';
 import { ReducedMotionManager } from '../../ReducedMotion';
+import { EasingNameSymbol } from '../../Easing';
 
 function getEasingFromConfig(config: CustomConfig): string {
-  const easingName =
-    config.easingV && config.easingV.name in WebEasings
-      ? (config.easingV.name as WebEasingsNames)
-      : 'linear';
+  if (!config.easingV) {
+    return `cubic-bezier(${WebEasings['linear'].toString()})`;
+  }
+
+  const easingName = config.easingV[EasingNameSymbol];
+
+  if (!(easingName in WebEasings)) {
+    console.warn(
+      `[Reanimated] Selected easing is not currently supported on web.`
+    );
+
+    return `cubic-bezier(${WebEasings['linear'].toString()})`;
```

Comment:
extract this to a separate string since its repeated

Reply:
It will be extracted into separate function in #6239.

Reply:
I really wanted to merge `CurvedTransition` last week, but it would be useless without this PR so I've decided to postpone it.